libraries/AP_ESC_Telem/AP_ESC_Telem_Backend.h

```diff
@@ -50,6 +55,11 @@ class AP_ESC_Telem_Backend {
         EDT2_STATUS = 1 << 8,
         EDT2_STRESS = 1 << 9,
 #endif
+#if AP_EXTENDED_ESC_TELEM_ENABLE
```

Comment:
```suggestion
#if AP_EXTENDED_ESC_TELEM_ENABLED
```

Reply:
modified to AP_EXTENDED_ESC_TELEM_ENABLED